README.md

```diff
@@ -1 +1,177 @@
-This is the meal website, consiming Meal API
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>To-Do List</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# Meal Webapp <a name="about-project"></a>
+
+**Meal Webapp** is a HTML, CSS and JavaScript project that displays a friendly interface for displaying Meal information from MealDB API, and manage Likes and Comments information from every Meal, identified by idMeal. You would see the wireframe design implemeted with the use of Bootstrap and little custom CSS classes. This is the capstone project required for Microverse Module 2.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+    <li><a href="https://www.atlassian.com/es/git/tutorials/comparing-workflows/gitflow-workflow">Gitflow</a></li>
+    <li><a href="https://keepcoding.io/blog/que-es-un-linter-en-programacion/">Linters</a></li>
+    <li><a href="https://webpack.js.org/">WebPack</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.themealdb.com/api.php">TheMealDB</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Webpack implementation for project development**
+- **HTML template to render the main page**
+- **Dynamic rendering of static Meals, Likes and comments with the use of arrays of JSON objects**
+- **Bootstrap and CSS styling**
+- **Pop-up to show single meal**
+- **Add Likes & Comments, preserving data**
+- **Data from meals retrived from TheDataBaseAPI**
+- **Data from likes & comments retrived from InvolvementAPI**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo!](https://alexansaa.github.io/Meals/)
```

Comment:
- [ ] Kindly make sure your live demo link is up-to-date with the latest changes in the app. Or remove its section from the README file for a professional repo. 🙏